From ca32fc17184e4b7b2942105067f4038191a7c4d6 Mon Sep 17 00:00:00 2001
From: faust <faustin@fala.red>
Date: Fri, 21 Dec 2018 14:52:03 -0800
Subject: [PATCH] Insane line length detected (#4813)

Summary:
Lintian, the Debian package checker complains about
insane-line-length-in-source-file.
Line length is 278 characters (>256)

Please see here the error:
https://salsa.debian.org/mariadb-team/mariadb-10.3/-/jobs/95739
Pull Request resolved: https://github.com/facebook/rocksdb/pull/4813

Differential Revision: D13539183

Pulled By: miasantreble

fbshipit-source-id: 28ad31d1bf23a076b9e4fc9ff62fb0b4c63a65f6
---
 docs/_includes/plugins/post_social_plugins.html | 9 ++++++++-
 docs/_includes/plugins/twitter_follow.html      | 9 ++++++++-
 docs/_includes/plugins/twitter_share.html       | 9 ++++++++-
 docs/_includes/social_plugins.html              | 9 ++++++++-
 4 files changed, 32 insertions(+), 4 deletions(-)

--- a/storage/rocksdb/rocksdb/docs/_includes/plugins/post_social_plugins.html
+++ b/storage/rocksdb/rocksdb/docs/_includes/plugins/post_social_plugins.html
@@ -31,4 +31,11 @@
    }(document, 'script', 'facebook-jssdk'));
 </script>
 
-<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+<script>!function(d,s,id){
+  var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';
+  if(!d.getElementById(id)){
+  js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';
+  fjs.parentNode.insertBefore(js,fjs);
+  }
+  }(document, 'script', 'twitter-wjs');
+</script>
--- a/storage/rocksdb/rocksdb/docs/_includes/plugins/twitter_follow.html
+++ b/storage/rocksdb/rocksdb/docs/_includes/plugins/twitter_follow.html
@@ -2,4 +2,11 @@
   <a href="https://twitter.com/{{ include.href }}" class="twitter-follow-button pluginBlock" data-show-count="false">Follow @{{ include.href }}</a>
 </div>
 
-<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
\ No newline at end of file
+<script>!function(d,s,id){
+  var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';
+  if(!d.getElementById(id)){
+  js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';
+  fjs.parentNode.insertBefore(js,fjs);
+  }
+  }(document, 'script', 'twitter-wjs');
+</script>
--- a/storage/rocksdb/rocksdb/docs/_includes/plugins/twitter_share.html
+++ b/storage/rocksdb/rocksdb/docs/_includes/plugins/twitter_share.html
@@ -1,4 +1,11 @@
 <div class="pluginWrapper twitterSharePlugin">
   <a href="https://twitter.com/share" class="twitter-share-button" data-hashtags="{{ site.title| replace: ' ', '' }}">Tweet</a>
 </div>
-<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
\ No newline at end of file
+<script>!function(d,s,id){
+  var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';
+  if(!d.getElementById(id)){js=d.createElement(s);
+  js.id=id;js.src=p+'://platform.twitter.com/widgets.js';
+  fjs.parentNode.insertBefore(js,fjs);
+}
+}(document, 'script', 'twitter-wjs');
+</script>
--- a/storage/rocksdb/rocksdb/docs/_includes/social_plugins.html
+++ b/storage/rocksdb/rocksdb/docs/_includes/social_plugins.html
@@ -21,4 +21,11 @@
   fjs.parentNode.insertBefore(js, fjs);
 }(document, 'script', 'facebook-jssdk'));</script>
 
-<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+<script>!function(d,s,id){
+  var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';
+  if(!d.getElementById(id)){
+  js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';
+  fjs.parentNode.insertBefore(js,fjs);
+  }
+  }(document, 'script', 'twitter-wjs');
+</script>
